playerctl/playerctl-cli.c

```diff
@@ -16,279 +16,359 @@
  * along with playerctl If not, see <http://www.gnu.org/licenses/>.
  *
  * Copyright © 2014, Tony Crisci
+ * Copyright © 2016, Jente Hidskes <hjdskes@gmail.com>
```

Comment:
Ultimately, I do not care so much. I just want recognition of my work, and nothing more. I have now created a CONTRIBUTORS file; it might now be needed as much now but perhaps in the future more people will want to add their names to it.